### DISCUSSION TITLE
GitPod Timed Out Capstone DA Telebot

### DISCUSSION THREAD

**Discussion by andrewanggoh** [teamalgoritma/community#624] [Q&A] [answered]
Saya menggunakan alternative GitPod untuk menjalankan app GitHub telebot saya. Lalu web GitPod saya close dan setelah beberapa saat (> 3menit), saya membuka untuk menjalankan programnya lagi. Ternyata muncul 'Timed Out' seperti screenshot dibawah. Kemudian, saya pilih 'Open Workspace' dan domain pada PORTS ter-default menjadi privacy. Untuk membuat domain menjadi public lagi, saya perlu untuk memilih 'Make Public' pada notifikasi yang muncul di kanan bawah atau pun secara manual seperti pada tutorial. Apakah hal tersebut memang biasa terjadi karena GitPod tidak men-deploy app continuously? atau ada hal lain yang perlu saya setting lagi? Terima kasih sebelumnya.

![Screenshot (2641)](https://user-images.githubusercontent.com/127578298/231317238-8ea14aa3-52db-4c75-b226-df8497c84a1c.png)

**Comment by VictorNugraha** [comment, marked as answer]
Hi @andrewanggoh,

Untuk pertanyaan yang diberikan, memang betul Gitpod tidak melakukan deployment secara contionous melainkan untuk beberapa saat saja (3 menit). Maka dari itu untuk menjalankan aplikasinya lagi, Mas Andrew harus melakukan proses deployment ulang pada aplikasi Gitpod.

Berikut jawaban yang dapat kami berikan. Semoga menjawab. 

Terima kasih.

**Reply by andrewanggoh** [reply]
baik, terima kasih kak 🙏
